src/main/webapp/assets/modules/common/securityService.js

```diff
@@ -146,22 +146,28 @@ techlooper.factory("securityService", function (apiService, $rootScope, $q, util
 
         var view = utils.getView();
         switch (view) {
-          case jsonValue.views.freelancerPostProject:
-            var lastPage = "/freelancer/post-project";
-          case jsonValue.views.employerDashboard:
-            var lastPage = "/employer-dashboard";
           case jsonValue.views.createEvent:
-            var lastPage = "/post-event";
+            //var lastPage = "/post-event";
+            var loginPage = "/user-type"
+            var role = "JOB_SEEKER";
```

Comment:
Expected a 'break' statement before 'case'.<br>'role' is defined but never used.